lor

```diff
@@ -11,6 +11,6 @@ Pending: (Failures listed here are expected and do not affect your suite's statu
 [36m     # ./spec/models/order_spec.rb:4
 [0m
 
-Finished in 4.65 seconds (files took 3.85 seconds to load)
-[33m51 examples, 0 failures, 2 pending[0m
+Finished in 4.72 seconds (files took 4.07 seconds to load)
+[33m58 examples, 0 failures, 2 pending[0m
```

Comment:
Looks like something to do with RSpec? Maybe add the file path to the gitignore?